.gitlab-ci.yml

```
@@ -3647,6 +3647,7 @@ deploy_staging_dsd:
   tags: [ "runner:main", "size:large" ]
   dependencies: []
   script:
+    - python3 -m pip install -r requirements.txt
```

The user we run as in `gitlab_agent_deploy` doesn't have admin rights

```suggestion
    - python3 -m pip install --user -r requirements.txt
```